hfuzz-signer/unsigned-message.rs

```diff
@@ -1,8 +1,5 @@
-use core::convert::TryFrom;
-use filecoin_signer::api::{MessageTx, MessageTxAPI, MessageTxNetwork, UnsignedMessageAPI};
-use forest_message::UnsignedMessage;
-
 fn main() {
+    /* FIXME
```

Comment:
maybe we should reinstate these if it's applicable so we don't lose fuzzing coverage

Reply:
I will create an issue to have this cover. And as mentioned we are now reusing directly the fvm_shared structures.